kotlinx.fuzz.examples/kotlinx.io/src/test/kotlin/org/plan/research/BuffersKt.kt

```diff
@@ -13,9 +13,9 @@ import org.plan.research.Constants.MAX_OPERATIONS_NUMBER
 import kotlin.test.assertEquals
 
 object BuffersKt {
-    val manyBytes = SystemFileSystem.source(Path("data.bin")).buffered().readByteString()
+    val manyBytes by lazy {org.plan.research.utils.largeByteString}
```

Comment:
Code style comment:
It is better to keep the spacing consistent everywhere. In our project we try to separate `{` and `}` with spaces from both sides (and it is also part of standard Kotlin formatting).

You can also use IJ auto-format feature to keep everything consistent with the default code style